lib/iris/tests/results/analysis/rotated_pole.0.rlat.json

```diff
@@ -1 +1 @@
-{"std": 3.7195861803241614, "min": -5.593200206756592, "max": 8.72130012512207, "shape": [93, 75], "masked": false, "mean": 1.6348200228305594}
\ No newline at end of file
+{"std": 3.7195863723754883, "min": -5.5932002067565918, "max": 8.7213001251220703, "shape": [93, 75], "masked": false, "mean": 1.6348202228546143}
```

Comment:
I don't consider it fudging as long as the value used remains appropriate.
I know that was not always the case for our image comparison tolerance, [where it got stupidly huge for a while...](https://github.com/SciTools/iris/pull/958/commits/36bc69d699cc49be81fc280ff795c46eb2eca7a3) !
The default value of 1e-7 is designed to cope with float32 rounding errors, but can easily be a bit fine for testing other calculated values, like coordinate projections.
For instance, I just encountered some other tests failing at that level when I was messing about with different dask chunkings.